shared/src/utils/random.ts

```diff
@@ -45,7 +54,7 @@ export default class Random {
   }
 
   /**
-   * Returns a random integer.
+   * From the uniform distribution on [min, max].
```

Comment:
Kommentar vervollständigen zu `Returns a random integer from the uniform distribution on [min, max].`